docs/topics/ref_new-features-7-1-0.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+//docs/release_notes-7.1.0/master.adoc
+
+
+:_newdoc-version: 2.18.2
+:_template-generated: 2024-07-01
+
+:_mod-docs-content-type: REFERENCE
+
+[id="new-features-7-1-0_{context}"]
+= New features
+
+This section provides the new features and improvements of the {ProductFullName} 7.1.0.
+
+.Support for analyzing applications managed with Gradle has been added to {ProductShortName} 
+
+Previously, you could use the {ProductFullName} to analyze Java applications managed only with Maven. With this update, {ProductShortName} can also extract dependencies from Gradle projects. As a result, you can now analyze applications that use Gradle instead of Maven. 
+
+:FeatureName: Support for analyzing applications managed with Gradle
+
+include::../snippets/developer-preview-admonition.adoc[]
+
+
+
+.Support for analyzing .NET applications has been added to {ProductShortName}
+
+In {ProductFullName} 7.1.0, you can use MTA to analyze Windows-only .NET framework applications to aid migration from version 4.5 or later to multi-platform .NET 8.0 running on OpenShift Container platform. This feature is only available in the command-line interface (CLI).
+
+:FeatureName: Support for analyzing .NET applications
+
+include::../snippets/developer-preview-admonition.adoc[]
+
+
+
+.Support for languages other than Java
+
+In {ProductFullName} 7.1.0, you can use {ProductShortName} to analyze .NET applications written in languages other than Java. To run analysis on .NET applications written in languages other than Java, add a custom rule set and do not specify a target language.
+
+:FeatureName: Support for languages other than Java
+
+include::../snippets/developer-preview-admonition.adoc[]
+
+
+
+.Assessment and review statuses are now displayed in the {ProductShortName} UI
+
+In earlier releases of {ProductShortName}, the status of the assessment and the review was only displayed for applications. With this update, the status of the archetype assessment and review processes is displayed in the {ProductFullName} user interface (UI):
+
+* The assessment statuses are the following:
+** Completed: All required assessments completed.
+** InProgress: The assessment process is in progress.
+** NotStarted: The assessment process has not been started.
+* The review statuses are the following:
+** Completed: A review exists.
+** NotStarted: The review process has not been started.
+
+
+
+.New Insights feature has been added
+
+Tagging rules that earlier generated tags and showed the presence of technology also generate insights now and show the location of code. While Insights do not impact the migration, they contain useful information about the technologies used in the application and their usage in the code. Insights do not have an effort and category assigned but might have a message and tag. You can view Insights in the Static report under the Insights tab.
```

Comment:
insight or **I**nsight

Reply:
Insights, I guess, but it's better to reconfirm with the responsible writer.